minvws/nl-kat-coordination
# Feature/improve rename bulk modal

### Changes

I've added the reset button to give the user the opportunity to discard any changes made to a reports name with one click. This was already implemented in the create report flow, so I went from there.

### Issue link

Closes #3873 

### Demo

https://github.com/user-attachments/assets/fafee7ff-28d4-41a7-9123-dbaacc6c0e1c


### QA notes

Select one (or more) reports and choose "Rename" in the "actions" dropdown. Edit the name of one or more reports in the modal. After the cursor leaves the input field a "back" arrow should be shown right of the input. Clicking

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
